src/main/java/edu/ksu/canvas/attendance/repository/OauthTokenRepository.java

```diff
@@ -0,0 +1,11 @@
+package edu.ksu.canvas.attendance.repository;
+
+import edu.ksu.canvas.attendance.entity.lti.OauthToken;
+import org.springframework.data.repository.CrudRepository;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public interface OauthTokenRepository extends CrudRepository<OauthToken, Long> {
+
+    OauthToken findByEIDAndApplicationName(String eID, String ApplicationName);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this local variable to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ome.ksu.edu/coding_rules#rule_key=squid%3AS00117)